README.md

```diff
@@ -158,8 +162,28 @@ As such it works with any executable that supports the following simple protocol
 
 Repeat forever:
 - read one line from standard input (file descriptor 0)
-- parse the line as a json object
-- execute the action, logging in standard output and in standard error (file descriptor 1 and 2)
+- parse the line as a json object, that will be in format:
+
+```{
+ "value": JSON,
+ "namespace": String,
+ "action_name": String,
+ "api_host": String,
+ "api_key": String,
+ "activation_id": String,
+ "deadline": Number
+}```
+
+Note that if you use libraries, those will expect the values in enviroment variables:
+
```

Comment:
Same comment here, this set is a base set of environment variables but more environment variables should be allow to be passed as additional fields.